ui/src/_components/CategoryPicker.svelte

```diff
@@ -0,0 +1,22 @@
+<script>
+    import GenreCard from './GenreCard.svelte'
+</script>
+
+<h2>Pick your category:</h2>
+<div class="grid grid-cols-4 gap-7 -mx-20 my-10 place-self-stretch">
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+    <GenreCard>Genre</GenreCard>
+</div>
```

Comment:
add a comment that this code scope will be changed when we will have an API route for genres